docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
-- [Imgix](https://www.imgix.com): `loader: 'imgix'`
-- [Cloudinary](https://cloudinary.com): `loader: 'cloudinary'`
-- [Akamai](https://www.akamai.com): `loader: 'akamai'`
-- Custom: `loader: 'custom'` use a custom cloud provider by implementing the [`loader`](/docs/api-reference/next/image.md#loader) prop on the `next/image` component
-- Default: Works automatically with `next dev`, `next start`, or a custom server
-
-If you need a different provider, you can use the [`loader`](/docs/api-reference/next/image.md#loader) prop with `next/image`.
-
-> The `next/image` component's default loader is not supported when using [`next export`](/docs/advanced-features/static-html-export.md). However, other loader options will work.
-
-> The `next/image` component's default loader uses [`squoosh`](https://www.npmjs.com/package/@squoosh/lib) because it is quick to install and suitable for a development environment. When using `next start` in your production environment, it is strongly recommended that you install [`sharp`](https://www.npmjs.com/package/sharp) by running `yarn add sharp` in your project directory. This is not necessary for Vercel deployments, as `sharp` is installed automatically.
+## Image Sizing
 
-## Caching
+One of the ways that images most commonly hurt performance is through *layout shift*, where the image pushes other elements around on the page as it loads in. This performance problem is so annoying to users that it has its own Core Web Vital, called [Cumulative Layout Shift](https://web.dev/cls/).  The way to avoid image-based layout shifts is to [always size your images](https://web.dev/optimize-cls/#images-without-dimensions). This allows the browser to reserve precisely enough space for the image before it loads.
 
-The following describes the caching algorithm for the default [loader](#loader). For all other loaders, please refer to your cloud provider's documentation.
+Because `next/image` is designed to guarantee good performance results, it cannot be used in a way that will contribute to layout shift, and **must** be sized in one of three ways:
+1) Automatically, using a [static import](#local-images)
+2) Explicitly, by including a `height` **and** `width` property
+3) By using the `fill` layout mode, which causes the image to expand to fill its parent element. 
 
-Images are optimized dynamically upon request and stored in the `<distDir>/cache/images` directory. The optimized image file will be served for subsequent requests until the expiration is reached. When a request is made that matches a cached but expired file, the cached file is deleted before generating a new optimized image and caching the new file.
+>### What if I don't know the size of my images?
+>
+>If you are accessing images from a source without knowledge of the images' sizes, there are several things you can do:
```

Comment:
Something seems wrong here. Why is this in a `<blockquote>`? What is it quoting?

Reply:
Do we have a preferred styling for something that's essentially an [aside](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside)? I feel like it doesn't quite fit into the main flow of the article, since it's kind of a mini-guide about a specific issue commonly faced by developers.

Reply:
I figured blockquote was the way to go because it's used throughout the docs for things which are more callouts than actual quotes.

Reply:
We can try it out and see how it looks once rendered